predicators/spot_utils/spot_utils.py

```diff
@@ -344,11 +354,25 @@ def get_objects_in_view_by_camera(
             # Next, convert the keys of this dictionary to be april
             # tag id's instead.
             for source_name, obj_pose_dict in detic_sam_pose_results.items():
-                viewable_obj_poses: Dict[int, Tuple[float, float, float]] = {}
                 for k, v in obj_pose_dict.items():
                     viewable_obj_poses[obj_name_to_apriltag_id[
                         vision_prompt_to_obj_name[k]]] = v
                 tag_to_pose[source_name].update(viewable_obj_poses)
+        elif mode == "cv2":
+            cv2_results = self.get_cv2_pose_from_imgs(
+                object_names=CV2_OBJ_LIST,
+                rgb_image_dict=rgb_image_dict,
+                depth_image_dict=depth_image_dict,
+                rgb_image_response_dict=rgb_image_response_dict,
+                depth_image_response_dict=depth_image_response_dict)
+            # Next, convert the keys of this dictionary to be april
+            # tag id's instead.
+            for source_name, obj_pose_dict in cv2_results.items():
+                for k, v in obj_pose_dict.items():
+                    viewable_obj_poses[obj_name_to_apriltag_id[k]] = v
+                tag_to_pose[source_name].update(viewable_obj_poses)
```

Comment:
whoa - i thought we agreed that we would never use CV2 to get object positions??